specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-06-01/containerregistry_build.json

```diff
@@ -0,0 +1,2578 @@
+          "format": "int32",
+          "description": "Time in seconds that the token remains valid",
+          "type": "integer"
+        }
+      }
+    },
+    "TaskUpdateParameters": {
+      "description": "The parameters for updating a task.",
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/IdentityProperties",
+          "description": "Identity for the resource."
+        },
+        "properties": {
+          "$ref": "#/definitions/TaskPropertiesUpdateParameters",
+          "description": "The properties for updating a task.",
+          "x-ms-client-flatten": true
+        },
+        "tags": {
+          "description": "The ARM resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "TaskPropertiesUpdateParameters": {
+      "description": "The properties for updating a task.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "The current status of task.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "TaskStatus",
+            "modelAsString": true
+          }
+        },
+        "platform": {
+          "$ref": "#/definitions/PlatformUpdateParameters",
+          "description": "The platform properties against which the run has to happen."
+        },
+        "agentConfiguration": {
+          "$ref": "#/definitions/AgentProperties",
+          "description": "The machine configuration of the run agent."
+        },
+        "timeout": {
+          "format": "int32",
+          "description": "Run timeout in seconds.",
+          "type": "integer"
+        },
+        "step": {
+          "$ref": "#/definitions/TaskStepUpdateParameters",
+          "description": "The properties for updating a task step."
+        },
+        "trigger": {
+          "$ref": "#/definitions/TriggerUpdateParameters",
+          "description": "The properties for updating trigger properties."
+        },
+        "credentials": {
+          "$ref": "#/definitions/Credentials",
+          "description": "The parameters that describes a set of credentials that will be used when this run is invoked."
+        }
+      }
+    },
+    "PlatformUpdateParameters": {
+      "description": "The properties for updating the platform configuration.",
+      "type": "object",
+      "properties": {
+        "os": {
+          "description": "The operating system type required for the run.",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "OS",
+            "modelAsString": true
+          }
+        },
+        "architecture": {
+          "description": "The OS architecture.",
+          "enum": [
+            "amd64",
```

Comment:
Is it possible to share it with enum on line 943?

Reply:
good catch! i missed adding it to that one. Both of these needs to be still different properties tho.
`PlatformProperties` is passed while creating a Task.
`PlatformUpdateProperties` is passed while updating a Task.